community/editorconfig/APKBUILD

```diff
@@ -2,39 +2,41 @@
 # Maintainer: Paul Morgan <jumanjiman@gmail.com>
 pkgname=editorconfig
 _pkgname=$pkgname-core-c
-pkgver=0.12.2
+pkgver=0.12.3
 pkgrel=0
 pkgdesc="EditorConfig core library written in C (for use by plugins supporting EditorConfig parsing)"
+options="!check" # Testsuite only via git checkout
```

Comment:
The test suite is in a separate git repo added as a submodule. The repo for that submodule does seem to have at least some [releases][], so it might be possible to add that as a separate source?

[releases]:https://github.com/editorconfig/editorconfig-core-test/releases

Reply:
Done